tests/rt.conf

```diff
@@ -2,20 +2,24 @@
 # CCPP PROD tests                                                                                                                                                                     #
 #######################################################################################################################################################################################
 
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                           | standard    | hera.intel     | fv3         |
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                           | standard    | cheyenne.intel | fv3         |
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                           | standard    | cheyenne.gnu   | fv3         |
+COMPILE | 32BIT=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                   | standard    | hera.intel     | fv3         |
+COMPILE | 32BIT=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                   | standard    | cheyenne.intel | fv3         |
+COMPILE | 32BIT=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                   | standard    | cheyenne.gnu   | fv3         |
 
 RUN     | fv3_ccpp_gfs_v15p2                                                                                                             | standard    |                | fv3         |
+RUN     | fv3_ccpp_gfs_v15p2_coldstart                                                                                                   | standard    |                |             |
+RUN     | fv3_ccpp_gfs_v15p2_restart                                                                                                     | standard    |                |             | fv3_ccpp_gfs_v15p2_coldstart
```

Comment:
@DusanJovic-NOAA thanks for the note, I think I added the coldstart dependency as required for both v15p2 and v16beta